movement/watch_faces/complication/alarm_face.c

```diff
@@ -268,7 +306,20 @@ bool alarm_face_loop(movement_event_t event, movement_settings_t *settings, void
             } else if (state->setting_state == alarm_setting_idx_minute) {
                         state->alarm[state->alarm_idx].minute = (state->alarm[state->alarm_idx].minute + 1) % 60;
             } else _abort_quick_ticks(state);
-        } else if (!state->is_setting) break; // no need to do anything when we are not in settings mode and no quick ticks are running
+        } else if (!state->is_setting) {
+            if (_wait_ticks >= 0 && _wait_ticks <= INT8_MAX) _wait_ticks++;
```

Comment:
_wait_ticks will always be <= INT8_MAX. I think you can remove it and just keep the first half of the conditional.

Reply:
unless you want to guarantee that _wait_ticks never goes negative here, which would obviously be solved by changing the `<=` to `<`

Reply:
You are right, this is ought to be a `<` instead of `<=`. Thanks for finding this. This bug will not cause any trouble in real life, though. The erroneous line means that if you hold the alarm button for more than 255 seconds, the logic for jumping back to alarm slot no. 1 will be triggered again. But, as the wearer will already be sitting on alarm slot 1, nothing will happen. Nevertheless, I will fix this in one of the next PRs.